src/main/java/com/laytonsmith/core/functions/InventoryManagement.java

```diff
@@ -821,7 +818,13 @@ public Construct exec(Target t, Environment environment, Construct... args) thro
         }
 
         private boolean match(MCItemStack is, MCItemStack iis){
-            return (is.getTypeId() == iis.getTypeId() && is.getData().getData() == iis.getData().getData());
+            if(is.getData() == null && iis.getData() == null) {
+                return is.getTypeId() == iis.getTypeId();
```

Comment:
Since you're changing this now outdated line, I'd HIGHLY advise switching to getType(), as I'm going to have to change it to that anyway because handling items by numeric IDs is no longer guaranteed, and thus there is not even an equivalent function in Sponge.